### ISSUE TITLE
Design: MoFo design vision, branding & process

### ISSUE THREAD

**Issue by xmatthewx** [MozillaFoundation/MoFoLondon#1]
Details coming soon.

**Comment by xmatthewx** [comment]
@lovegushwa – post any question/ideas about Brand/Identity related things here. Connect with @ldecoursy to win. 

Thoughts: 
 - wait for moco to schedule things?
 - propose a timeslot to moco?
 - propose an elective to lead a thing (see email)

Beyond this, if we want any design team time, we should file that as a separate issue.

**Comment by OpenMatt** [comment]
@xmatthewx can you document a goal or outcome you want from this?

**Comment by xmatthewx** [comment]
@lovegushwa @HPaulJohnson – any news of MoCo plans for brand identity discussions? If there's nothing clear yet for us to hook into, we should devise a plan of our own. Kristina, Jordan, and Paul could facilitate a MoFo identity discussion.

**Comment by lovegushwa** [comment]
@xmatthewx nothing concrete yet from brand folks. I'm going to close this issues and open a MOFO design team session ticket.

**Comment by lovegushwa** [comment]
## Description:

Part 1: Share design vision: Each designer gives a lightning talk about a design hero, where they see design going, and from these two things a quality, attribute or project Mozilla design should embrace. Our goal here is to expand our design vision as a group, and highlight ideas that could impact design across MOCO and MOFO.

Part 2: Mozilla rebrand hack: MOCO branding has defined Mozilla brand attributes. Do they align with the design attributes we defined in 2015? Let’s evaluate and leave with a shared set of attributes we express every time we design a product or project.

Part 3: MOFO design repo: Will start with a postmortem on collaborative design studio efforts since Whistler. What worked, what didn't, what do we want to do next. Big question, how do we want to work in the new org structure. (MLN / Advocacy) ?  What do we expect from each other moving forward?

### Time requirements:

Three hours

### Space requirements:

A space with enough room for 7 participants, large notepad or whiteboard.

### Required attendees:

@xmatthewx @vazquez @sabrinang @flukeout @natalieworth @kristinashu @lovegushwa

### Dependencies: (Other sessions that need to happen before this one)

NA

**Comment by lovegushwa** [comment]
CC @OpenMatt

**Comment by xmatthewx** [comment]
cc @kristinashu @sabrinang @natalieworth @flukeout - let's continue to discuss!!

**Comment by natalieworth** [comment]
I feel a postmortem and discussion/ documentation of a future shared vision could be beneficial. I wasn't here for Whistler, but I think looking towards our expectations to how MLN and Advocacy will work together under MoFo would help move us forward (and maybe how we can continue collaborating through the design repo?). We could possibly reflect on our principles/ branding/ org structure and then give thought to where we see ourselves going (either through a small design lightning talk or even just speculating and hopes for where MoFo design could go).

**Comment by natalieworth** [comment]
Maybe we could somehow incorporate a design jam or hack into this so we could creatively document what we get out of our time together?

**Comment by xmatthewx** [comment]
Our lists for betterfaster follow up might be useful for our chats in London: https://github.com/MozillaFoundation/Design/issues/161

**Comment by kristinashu** [comment]
More things we want to discuss: 
![design-london-notes](https://cloud.githubusercontent.com/assets/1068783/16671767/3f906ba6-4456-11e6-9566-948b3d294eaf.jpg)
